server/main_server/src/main.cpp

```diff
@@ -30,6 +32,7 @@
 #include <boost/stacktrace.hpp>
 
 #include <fmt/format.h>
+#include <fmt/compile.h>
```

Comment:
Unless the include order was intentional...
```suggestion
#include <fmt/compile.h>
#include <fmt/format.h>
```